spec/unit/provider/user/hpux_spec.rb

```diff
@@ -49,4 +50,23 @@
       provider.password.must == "foopassword"
     end
   end
+
+  context "Check for trusted computing" do
+  before :each do
+    provider.stubs(:command).with(:modify).returns '/usr/sam/lbin/usermod.sam'
+  end
+
+  it "Should add modprpw to modifycmd if Trusted System" do
+    resource.stubs(:allowdupe?).returns true
+    provider.stubs(:exec_getprpw).with('root','-m uid').returns('uid=0')
```

Comment:
@haus pointed out that we should use provider.expects(:exec_getprpw)... instead of stubs, as it adds the expectation that the method will be called. It'll still stub the method.